perllib/FixMyStreet/App/Form/Waste/Bulky.pm

```diff
@@ -216,24 +216,8 @@ has items_master_list => (
 );
 
 sub _build_items_master_list {
-    $_[0]->c->cobrand->call_hook('bulky_items_master_list');
-}
-
-has items_by_category => (
-    is      => 'ro',
-    isa     => 'HashRef',
-    lazy    => 1,
-    builder => '_build_items_by_category',
-);
-
-sub _build_items_by_category {
-    my $self = shift;
-
-    my %hash;
-    for my $item ( @{ $self->items_master_list } ) {
-        push @{ $hash{ $item->{category} } }, $item->{name};
-    }
-    return \%hash;
+    [ sort { $a->{name} cmp $b->{name} }
```

Comment:
One tiny improvement would be to do a case-insensitive search (I noticed "e-Scooters" was at the bottom of my list):

```suggestion
    [ sort { lc($a->{name}) cmp lc($b->{name}) }
```

Reply:
👍